src/schema/submissions.ts

```diff
@@ -0,0 +1,113 @@
+import { Post, Submission, User } from './../entity';
+import { IResolvers } from 'graphql-tools';
+import { traceResolvers } from './trace';
+import { Context } from '../Context';
+import { getDiscussionLink, isValidHttpUrl } from '../common';
+import { ValidationError } from 'apollo-server-errors';
+import { GQLEmptyResponse } from './common';
+
+interface GQLArticleSubmission {
+  url: string;
+}
+
+interface GQLSubmissionAvailability {
+  hasAccess: boolean;
+  limit: number;
+  todaySubmissionsCount: number;
+}
+
+export const DEFAULT_SUBMISSION_LIMIT = '3';
+
+export const typeDefs = /* GraphQL */ `
+  type SubmissionAvailability {
+    hasAccess: Boolean!
+    limit: Int!
+    todaySubmissionsCount: Int!
+  }
+
+  extend type Query {
+    """
+    Information regarding the access of submitting community links
+    """
+    submissionsAndLimit: SubmissionAvailability!
```

Comment:
I think this name makes more sense as this is the name of the object as well

```suggestion
    submissionAvailability: SubmissionAvailability!
```